BLE_GAP/source/main.cpp

```diff
@@ -577,42 +434,17 @@ class GapDemo : private mbed::NonCopyable<GapDemo>, public ble::Gap::EventHandle
         ble::connection_handle_t connectionHandle,
         uint16_t txSize,
         uint16_t rxSize
-    ) {
+    )
+    {
         printf(
             "Data length changed on the connection %d.\r\n"
             "Maximum sizes for over the air packets are:\r\n"
             "%d octets for transmit and %d octets for receive.\r\n",
-            connectionHandle,
-            txSize,
-            rxSize
+            connectionHandle, txSize, rxSize
         );
     }
 
```

Comment:
What do you think of showcasing: 
- onAdvertisingStart
- onAdvertisingEnd
- onScanTimeout

Reply:
Some are already there, I added onAdvertisingStart